tests/rules/no-important.test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview Tests for no-important rule.
+ * @author Yann Bertrand
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/no-important.js";
+import css from "../../src/index.js";
+import { RuleTester } from "eslint";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		css,
+	},
+	language: "css/css",
+});
+
+ruleTester.run("no-important", rule, {
+	valid: [
+		"a { color: red; }",
+		"a { color: red; background-color: blue; }",
+		"a { color: red; transition: none; }",
+		"body { --custom-property: red; }",
```

Comment:
I'd also add a test with just the word "important":

```suggestion
		"body { --custom-property: red; }",
		"body { --custom-property: important; }",		
```